<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout" rowstretch="10,2">
      <property name="horizontalSpacing">
       <number>15</number>
      </property>
      <property name="verticalSpacing">
       <number>13</number>
      </property>
      <item row="1" column="1">
       <widget class="QFrame" name="frame_label_area">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QTextEdit" name="textEdit_show_label"/>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QFrame" name="frame_image_area">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="horizontalSpacing">
            <number>35</number>
           </property>
           <item row="0" column="0">
            <widget class="QLabel" name="label_original_image">
             <property name="text">
              <string>original_image</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="label_predicted_image">
             <property name="text">
              <string>predicted_image</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QFrame" name="frame_setup_area">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="3,5,1">
         <property name="spacing">
          <number>15</number>
         </property>
         <item>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_source">
             <property name="text">
              <string>Source </string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_model">
             <property name="text">
              <string>Model </string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLabel" name="label_winsize">
             <property name="text">
              <string>Window-size</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_set_winside"/>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="comboBox_choose_model">
             <item>
              <property name="text">
               <string>Baseline model</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Sliding window</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButton_choose_source">
             <property name="text">
              <string>choose source</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="label_conf">
               <property name="text">
                <string>Confidence Threshold</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QSlider" name="slider_conf">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_conf_value">
                 <property name="text">
                  <string>0.25</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="label_iou">
               <property name="text">
                <string>IoU Threshold</string>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_4">
               <item>
                <widget class="QSlider" name="slider_iou">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_iou_value">
                 <property name="text">
                  <string>0.65</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QPushButton" name="pushButton_run_prediction">
             <property name="text">
              <string>Predict</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_next_image">
             <property name="text">
              <string>Next image</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButton_previous_image">
             <property name="text">
              <string>Prev image</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1080</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExample"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuMode">
    <property name="title">
     <string>Mode</string>
    </property>
    <addaction name="actionmode1"/>
    <addaction name="actionmode2"/>
    <addaction name="separator"/>
    <addaction name="actionmode3"/>
    <addaction name="actionmode4"/>
    <addaction name="actionmode5"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="actionDocument"/>
    <addaction name="separator"/>
    <addaction name="actionAbout_Project"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMode"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExample">
   <property name="text">
    <string>Example</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionmode1">
   <property name="text">
    <string>mode1</string>
   </property>
  </action>
  <action name="actionmode2">
   <property name="text">
    <string>mode2</string>
   </property>
  </action>
  <action name="actionmode3">
   <property name="text">
    <string>mode3</string>
   </property>
  </action>
  <action name="actionmode4">
   <property name="text">
    <string>mode4</string>
   </property>
  </action>
  <action name="actionmode5">
   <property name="text">
    <string>mode5</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionDocument">
   <property name="text">
    <string>Document</string>
   </property>
  </action>
  <action name="actionAbout_Project">
   <property name="text">
    <string>About Project</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
